infra/app/app-config/prod.tf

```diff
@@ -16,4 +16,10 @@ module "prod_config" {
   service_cpu                    = 1024
   service_memory                 = 4096
   service_desired_instance_count = 3
+
+  # Enables ECS Exec access for debugging or jump access.
+  # ⚠️ Warning! It is not recommended to enable this in a production environment.
```

Comment:
Possibly related to the larger design decision I commented about in the service, but FWIW, we (OSRE) actually provide ECS jumphosts _specifically_ for production debugging purposes, as lower environments are open to the VPN.

Reply:
Given that this pattern is _instead of_ VPN, I could see setting up a separate jump host just for production. I can add that as a follow on issue.

Reply:
FWIW, grants.gov is also planning on using this specifically for debugging and basic ops work in production.

Reply:
I appreciate both data points. I'll modify the PR name to just "Add support for ECS Exec for debugging" then. I'll tag you both for further discussion of production access in the issue.

Reply:
Follow up issue: https://github.com/navapbc/template-infra/issues/595